Kickstarter-iOS/DataSources/CommentsDataSourceTests.swift

```diff
@@ -5,14 +5,16 @@ import Prelude
 import XCTest
 
 class CommentsDataSourceTests: XCTestCase {
-  let dataSource = CommentsDataSource()
-  let tableView = UITableView()
-  let commentsSection = CommentsDataSource.Section.comments.rawValue
-  let emptySection = CommentsDataSource.Section.empty.rawValue
+  private let commentsSection = CommentsDataSource.Section.comments.rawValue
+  private let dataSource = CommentsDataSource()
+  private let emptySection = CommentsDataSource.Section.empty.rawValue
+  private let tableView = UITableView()
+
+  private let templateComments = Comment.templates + [Comment.retryingTemplate, Comment.retrySuccessTemplate]
```

Comment:
Is there a cleaner syntax we can use for this? Maybe even create a template that has all these templates within it?

Reply:
I added this because it's only useful in the tests.